### DISCUSSION TITLE
FHIR standard

### DISCUSSION THREAD

**Discussion by didymu5** [supabase/supabase#14175] [Questions]
Has anyone used Supabase for a FHIR implementation? https://www.hl7.org/fhir/overview.html 
working on a healthcare non-profit solution.

**Comment by MihaiTudorP** [comment]
I was looking for the same thing, stumbled upon this, waiting for oher people's opinion. I already have a semi-working prototype developed with Supabase, but I don't really know about FHIR compliance, I need to look further and compare specs to see.

**Comment by Dokotela** [comment]
I'm the lead developer for [FHIR-FLI](https://github.com/fhir-fli) and I'd love if there was a FHIR option for Supabase. I started working on some basic structure to store [FHIR in Supabase](https://github.com/fhir-fli/fhir_supabase/tree/main/sql) earlier this year, but then it seemed a bit overwhelming for one person (database design is NOT my thing). It can obviously be done, there's even a whole working group called [SQL on FHIR](https://github.com/FHIR/sql-on-fhir-v2) led by the folks at [Health Samurai](https://www.health-samurai.io/articles/sql-on-fhir-an-inside-look), but I think it would need to be a group effort. If ya'll are still interested in working on it though, I'd definitely be interested in helping. (I think one of the issues I had at the time is Supabase wasn't HIPAA compliant yet, although they were working on it).

**Reply by raqibhayder** [reply]
@kiwicopple: Any plans of supporting the FHIR standard?

**Comment by Gil-Frenkel** [comment]
Well basically HAPI-FHIR is on top of PostgresSQL, and Supabase is also on top of PostgreSQL.. I don't see why it can't be straight forward..
